_layouts/base.html.slim

```diff
@@ -371,6 +371,7 @@
       / script src="#{site.base_url}/javascripts/searchpage.js"
       script src="#{site.base_url}/javascripts/a-b-testing.js"
       script src="#{site.base_url}/javascripts/product-forums.js"
+      script src="#{site.base_url}/javascripts/adaptive-placeholder.js"
```

Comment:
Since we're now using Drupal, changing this base layout will not do anything. New javascript files need to be added to the Gulpfile.